src/tests/calculate.test.js

```diff
@@ -28,21 +40,65 @@ describe('calculate', () => {
   });
 
   it('constructs the (next)number as you type if you pressed an operation button', () => {
-    let data = { total: '123', next: '', operation: '' };
+    let data = {
+      total: '123',
+      next: '',
+      operation: '',
+    };
     data = calculate(data, '+');
     expect(data.operation).toBe('+');
   });
 
   it('constructs the (next)number as you type if you pressed an operation button', () => {
-    let data = { total: '12', next: '3', operation: '+' };
+    let data = {
+      total: '12',
+      next: '3',
+      operation: '+',
+    };
     data = calculate(data, '=');
     expect(data.total).toBe('15');
   });
 
   it('constructs the (next)number as you type if you pressed an operation button', () => {
-    let data = { total: '', next: '3', operation: '' };
+    let data = {
+      total: '',
+      next: '3',
+      operation: '',
+    };
     data = calculate(data, '.');
     data = calculate(data, '2');
     expect(data.next).toBe('3.2');
   });
+
+  it('Should return a negative value', () => {
+    const result = calculate({ total: '6', next: '10', operation: '+' }, '=');
+    expect(result.total * 1).toEqual(16);
+    expect(result.next).toEqual(null);
+    expect(result.operation).toEqual('=');
+  });
+  it('Should return a sustrated value', () => {
+    const result = calculate({ total: '6', next: '10', operation: '-' }, '=');
+    expect(result.total * 1).toEqual(-4);
+    expect(result.next).toEqual(null);
+    expect(result.operation).toEqual('=');
+  });
+  it('Should return a multiplied value', () => {
```

Comment:
- [ ] It would be better if we write a unit test for the following `%`, `+/-`, `AC` as mentioned by the previous reviewers. This will ensure our application functions as expected